app/Http/Controllers/EventController.php

```diff
@@ -103,29 +103,30 @@ public function join(Request $request, Event $event)
 
     public function remove(Request $request,Event $event, Enrollment $enrollment)
     {
-        $edition = $request->input('edition');
+        if (Gate::allows('admin') || Gate::allows('staff', [$edition])) {
```

Comment:
this is useless, considering that the check was added at the middleware level

Reply:
this is different than the middleware though, the `can:admin` middleware only enforces `Gate::allows("admin")`